src/build.d

```diff
@@ -368,10 +422,6 @@ LtargetsLoop:
                 "TODO: info".writeln; // TODO
                 break;
 
-            case "unittest":
```

Comment:
So we can't run "build.d unittest" now?

Reply:
you still can, the `runDmdUnittest` dependency has the name set to `unittest`